docs/whitepaper.tex

```diff
@@ -247,38 +247,98 @@ \section{Tests}
 
 \todo{Should we write Apropos-Tx tests?}
 
-\subsection{Unit tests}
+\subsection{Components}
```

Comment:
Is there a reason this is all put into one subsection, vs. turning each of these subsubsections into its own subsection?

Reply:
I think I was planing to have more subsections but end up with just the components.